acceptance/README.md

```diff
@@ -1,257 +1,467 @@
+You can safely ignore the data associated with the key.
+Generally, this string will be in the format
+`{platform}{version}-{architecture}{role/s}`. You will most often use either the
+agent (a) or master (m) role, but you can find a list of available roles in
+[beaker hostgenerator](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/roles.rb).
+Multiple hosts in the string are separated with a dash(`-`). You must have at
+least one agent and at least one master.
+
+Be careful not to confuse the different host string formats. We have different
+tools that expect the host string to be in different forms. For example,
+`packaging_platform` is specific to how [Vanagon](https://github.com/puppetlabs/vanagon)
+parses that string.
+
+### hosts.yaml
+The rake tasks that run acceptance will by default create a hosts file and
+populate it using [beaker-hostgenerator][] using either the `HOSTS` environment
+variable or the default host string (currently `redhat7-64ma-windows2012r2-64a`).
+The automation assumes you are using the vmpooler hypervisor and an internal
+pooling api, which is only available to Puppet, Inc. employees. If you want to
+customize the hypervisor or the pooling api, you'll need to generate your own
+hosts file. You must pass in a valid host string to the `beaker-hostgenerator`
+command. See the previous section for more information on how to construct a
+valid host string.
+
+To customize the hypervisor, pass in `--hypervisor {hypervisor name}`. To set
+the pooling api, use `--global-config pooling_api={custom api}`. Only the
+vmpooler hypervisor uses the pooling api.
+
+The host string that is passed in is the same that you would use with the
+`HOSTS` environment variable. See the [previous section](##customizing-test-targets) on how to format this string.
+
+For example, if you were to run this command:
+```
+bundle exec beaker-hostgenerator redhat7-64ma-windows2012r2-64a --disable-default-role --osinfo-version 1 --hypervisor vmpooler --global-config pooling_api=http://customvmpooler/ > hosts.yaml
+```
+You would generate a file called `hosts.yaml` that contains something like this:
+```
+---
+HOSTS:
+  redhat7-64-1:
+    platform: el-7-x86_64
+    packaging_platform: el-7-x86_64
+    template: redhat-7-x86_64
+    hypervisor: vmpooler
+    roles:
+    - master
+    - agent
+  windows2012r2-64-1:
+    platform: windows-2012r2-64
+    packaging_platform: windows-2012-x64
+    ruby_arch: x64
+    template: win-2012r2-x86_64
+    hypervisor: vmpooler
+    roles:
+    - agent
+CONFIG:
+  nfs_server: none
+  consoleport: 443
+  pooling_api: http://customvmpooler/
+```
+To have the automation recognize and use your custom hosts file, you'll need to
+set the `HOSTS` environment variable to the hosts file. In the above example, we
+called this file `hosts.yaml`, so we will use `HOSTS=hosts.yaml` when running
```

Comment:
"we will use"

Do you mean I need to keep using/specifying it? Or that it will automatically be used now that I have just called it? Pretty sure it's the latter, but the we makes it uncertain.

This little blurb should also probably come earlier in the section, since this is important info.